frontend/packages/signless-transactions/src/components/create-session-modal/create-session-modal.tsx

```diff
@@ -87,39 +87,46 @@ function CreateSessionModal({ close }: Props) {
 
   return (
     <>
-      <Modal heading="Create Signless Session" close={close}>
+      <Modal heading="Enable Signless Session" close={close}>
         <ul className={styles.summary}>
-          <li>
+          <li className={styles['summary-item']}>
             <h4 className={styles.heading}>
               {storagePair ? 'Account from the storage:' : 'Randomly generated account:'}
             </h4>
-
+            <div className={styles.separator} />
             {pair && (
               <div className={styles.account}>
-                <Identicon value={pair.address} theme="polkadot" size={14} />
-                <span>{getVaraAddress(pair.address)}</span>
+                <span className={styles.value}>{shortenString(getVaraAddress(pair.address), 4)}</span>
+                <FileCopyIcon
+                  onClick={() => copyToClipboard({ value: getVaraAddress(pair.address), alert })}
+                  className={styles.copy}
+                />
               </div>
             )}
           </li>
 
-          <li>
+          <li className={styles['summary-item']}>
             <h4 className={styles.heading}>Voucher to issue:</h4>
-            <p>
-              {formattedIssueVoucherValue.value} {formattedIssueVoucherValue.unit}
+            <div className={styles.separator} />
+            <p className={styles.value}>
+              {formattedIssueVoucherValue.value.replace(/\.(.*\d{2})/, (match) => `.${match.slice(1, 3)}`)}
```

Comment:
Divide an util function for this, not quite obvious what we try to achieve there

Reply:
We need just to cut last two 00 from an balance value string so for example 12.0000 to be 12.00 
What should we better use instead of regular expressions?

Reply:
Same way just create an util function with an appropriate name

Reply:
And what about `12.0001` cases for example?

Reply:
> And what about `12.0001` cases for example?

I guess its also better to be discussed with designers